ocaml/runtime/ints.c

```diff
@@ -334,9 +334,14 @@ CAMLprim value caml_int32_format(value fmt, value arg)
   return caml_alloc_sprintf(format_string, Int32_val(arg));
 }
 
+int32_t caml_int32_of_string_unboxed(value s)
```

Comment:
This should also have `CAMLprim` or the bootstrap for the upstream build won't see the primitive (I think).

Reply:
Added `CAMLprim`